documentation/configuration_files/application.md

```diff
@@ -0,0 +1,38 @@
+# application.yml
+
+```yml
+# application name - only lowercase alphanumeric character(s) separated by a single hyphen are allowed
+name:
+
+# configuration for locally running the application
+local:
+  dependencies:
+    <id>: # each dependency needs an id
+      image: # docker image
+      environment-variables: # map of environment variables to pass to this dependency
+      secrets: # array of shell environment variables to pass through to this dependency
+      persist: # array of paths within the docker images you would like saved in a docker volume
+  environment-variables: # map of environment variables to pass to every service  
+  secrets: # array of shell environment variables to pass to all services
```

Comment:
Can we move `environment-variables` and `secrets` up before `dependencies`? Even though there's a `dependency.<id>.environment-variables` section it's a little unclear that the env vars are for the application and not the dependencies.

Reply:
I don't think its unclear as its not nested under dependencies. The comment should make it clear too as it says "to pass to every service".